proxy/operator_resource/resource_manager.py

```diff
@@ -97,6 +103,7 @@ def get_resource(self, chain_id: int | None) -> OpResourceModel:
                     owner=op_signer.owner,
                     holder_address=op_holder.address,
                     resource_id=op_holder.resource_id,
+                    chain_id=chain_id,
```

Comment:
Please check the input parameter `chain_id`
It can be None.
It is possible because the chain-id for a stuck tx can be undefined.

Reply:
Done.

I have assumed that when a resource does not have a `chain_id` then we should use the default `chain_id`.

Reply:
Done. Please just check if here if using the default is correct.